docs/api-reference/io-columns-clustering.md

```diff
@@ -3,5 +3,5 @@ The input features column data must be <xref:System.Single>. No label column nee
 
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
-| `Score` | <xref:System.Single> | The unbounded score that was calculated by the trainer to determine the prediction.|
-| `PredictedLabel` | <xref:System.Int32> | The cluster id predicted by the trainer.|
\ No newline at end of file
+| `Score` | <xref:System.Single> | The unbounded score that was calculated by the trainer to determine the prediction. |
```

Comment:
Is `unbounded` the right word here? Would `raw` work equally well or better?

Reply:
Isn't the Score column in clustering actually a vector of the distances from the cluster centers?

---
In reply to: [277476207](https://github.com/dotnet/machinelearning/pull/3520#discussion_r277476207) [](ancestors = 277476207)

Reply:
Will do `The distances of the given data point to all clusters' centriods.`

---
In reply to: [277750343](https://github.com/dotnet/machinelearning/pull/3520#discussion_r277750343) [](ancestors = 277750343,277476207)